<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simple hashtable interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="hash_tablewidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>480</width>
      <height>170</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_item_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter number...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_item_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add the entered number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_number_of_numbers_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add the entered number of random numbers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rmv_item_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rmv item</string>
    </property>
   </widget>
   <widget class="QLabel" name="collisions_num_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Collisions: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rmv_in_range_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rmv items w keys in range</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="from_edit">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>41</y>
      <width>45</width>
      <height>18</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>from...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="to_edit">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>41</y>
      <width>45</width>
      <height>18</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>to...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
